index.src.html

```diff
@@ -905,25 +905,37 @@ <h3 id=interface-reporting-observer>Interface {{ReportingObserver}}</h3>
 
   Each {{ReportingObserver}} object has these associated concepts:
     - A <dfn for=ReportingObserver>callback</dfn> function set on creation.
-    - A {{ReportingObserverOptions}} dictionary called
-      <dfn for=ReportingObserver>options</dfn>.
+    - A list of strings called <dfn for=ReportingObserver>types</dfn>.
```

Comment:
Why are we dropping definition of options? The fields you're defining here are members of the dictionary. 

FWIW, I think we should keep current text.